components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md

```diff
@@ -208,10 +208,173 @@ postgresql_private_ips = ["D1.D2.D3.D4","E1.E2.E3.E4","F1.F2.F3.F4"]
 - `opensearch_private_ips` Eg: ["192.0.2.1", "192.0.2.2", "192.0.2.2"]
 - `postgresql_private_ips` Eg: ["192.0.3.1", "192.0.3.2", "192.0.3.2"]
 
+## On-Premise Setup with AWS Managed Services.
+
+### Prerequisites
+
+- Follow the Prerequisites for On-Premise deployment. Click [here](https://docs.chef.io/automate/ha_onprim_deployment_procedure/#prerequisites).
+- You can exclude the creation of instances for OpenSearch and Postgresql as you will be using AWS OpenSearch and AWS RDS Postgresql.
```

Comment:
reframe the sentence. This deployment excludes the installation for PG and OS as we are using the AWS Managed services.